lib/dslash_wilson_clover_hasenbusch_twist_preconditioned.cu

```diff
@@ -153,10 +154,15 @@ namespace quda
 
     instantiate<WilsonCloverHasenbuschTwistPCNoClovInvApply>(out, in, U, A, a, b, x, parity, dagger, comm_override,
                                                              profile);
+  }
 #else
+  void ApplyWilsonCloverHasenbuschTwistPCNoClovInv(ColorSpinorField &, const ColorSpinorField &,
+                                                   const GaugeField &, const CloverField &, double, double,
+                                                   const ColorSpinorField &, int, bool, const int *, TimeProfile &)
+  {
     errorQuda("Clover dslash has not been built");
```

Comment:
```suggestion
    errorQuda("Clover Hasensbuch Twist dslash has not been built");
```

Reply:
Since this also appears in 1.1, I've added this fix to #1160 which will be merged into the 1.1 branch shortly, and then we can merge that into GK.